Mage.Sets/src/mage/cards/b/BillThePony.java

```diff
@@ -0,0 +1,93 @@
+package mage.cards.b;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.EntersBattlefieldTriggeredAbility;
+import mage.abilities.common.SimpleActivatedAbility;
+import mage.abilities.costs.common.SacrificeTargetCost;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.CreateTokenEffect;
+import mage.abilities.effects.common.ruleModifying.CombatDamageByToughnessEffect;
+import mage.constants.*;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.filter.FilterPermanent;
+import mage.filter.common.FilterControlledCreaturePermanent;
+import mage.filter.common.FilterControlledPermanent;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.filter.predicate.mageobject.MageObjectReferencePredicate;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.game.permanent.token.FoodToken;
+import mage.target.TargetPermanent;
+import mage.target.common.TargetControlledPermanent;
+
+/**
+ *
+ * @author Susucr
+ */
+public final class BillThePony extends CardImpl {
+
+    private static final FilterPermanent filter = new FilterControlledCreaturePermanent();
+    private static final FilterControlledPermanent foodFilter = new FilterControlledPermanent(SubType.FOOD, "a Food");;
+
+    public BillThePony(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{3}{W}");
+        
+        this.supertype.add(SuperType.LEGENDARY);
+        this.subtype.add(SubType.HORSE);
+        this.power = new MageInt(1);
+        this.toughness = new MageInt(4);
+
+        // When Bill the Pony enters the battlefield, create two Food tokens.
+        this.addAbility(new EntersBattlefieldTriggeredAbility(new CreateTokenEffect(new FoodToken(), 2)));
+
+        // Sacrifice a Food: Until end of turn, target creature you control assigns combat damage equal to its toughness rather than its power.
+        Ability ability = new SimpleActivatedAbility(
+            new BillThePonyEffect(),
+            new SacrificeTargetCost(new TargetControlledPermanent(foodFilter))
+        );
+        ability.addTarget(new TargetPermanent(filter));
```

Comment:
there's an existing static filter for this too

Reply:
Like with Shadowfax, I did forgot to push the changes I had locally.
So there is a commit from the past fixing the issue.